NebulaPatcher/Patches/Dynamic/Mecha_Patch.cs

```diff
@@ -15,5 +16,21 @@ public static void GameTick_Postfix(long time, float dt)
                 SimulatedWorld.OnDronesGameTick(time, dt);
             }
         }
+
+        // We can't do this as client as we won't be able to get_nearestPlanet() since we do not currently have all of the factory info
+        [HarmonyPrefix]
+        [HarmonyPatch(typeof(Mecha), nameof(Mecha.AddConsumptionStat))]
+        [HarmonyPatch(typeof(Mecha), nameof(Mecha.AddProductionStat))]
+        public static bool AddStat_Common_Prefix(ref PlanetFactory factory)
+        {
+            if(!SimulatedWorld.Initialized || LocalPlayer.IsMasterClient)
+                return true;
+
+            // TODO: Send packet to host to add stat?
```

Comment:
Please resolve

Reply:
@Mushroom Thoughts on how this should be addressed? I don't think this should be a holdup for getting this merged, it's a very insignificant feature for playing the game and I think the solutions to this are all kind of hacky

Reply:
Okay if it's not a blocker I'd say leave as is and create an issue for it instead